frontend/src/plugins/store/index.js

```diff
@@ -41,4 +42,6 @@ class StorePlugin {
 export let apiStore
 export let store
 
+export let storePlugin = new StorePlugin()
```

Comment:
this is for the auth.spec.js test.
As soon as i started mocking the router, store.replaceState was undefined...?

Reply:
I think I was able to simplify this: b33f67eed04033510d7cc2fcd4b59c7d1f4d6a38